tools/node.js

```diff
@@ -73,11 +73,13 @@ foam.CLASS({
     function execute() {
       var modelArgs = this.modelArgs;
       var X = this.__subContext__;
-      return X.arequire(this.modelId).then(function(model) {
-        return model.create(modelArgs, X).execute();
-      }).catch(function(err) {
-        console.log(err.stack);
-      });
+      var modelId = this.modelId;
+      return X.arequire(modelId)
+        .catch(console.log)
```

Comment:
This script is only to be used when running node.js so I don't think we need to worry about how this works in browsers.